### ISSUE TITLE
How do I cancel a benchmark from the phoromatic web interface?

### ISSUE THREAD

**Issue by tstellarAMD** [phoronix-test-suite/phoronix-test-suite#57]
Hi,

I kicked off a new benchmark on one of my servers using the "benchmark" tab in phoromatic.  The benchmark now seems to be running on an infinite loop, meaning as soon as the benchmark has completed it starts all over again.

Is there any way to cancel this job?

**Comment by michaellarabel** [comment]
Hmmmm, is the test continuing to keep failing and thus repeating the ticket? That'd what would come to mind initially, is that if all the tests keep failing I think at the moment I don't have a check in place to prevent it from repeating (there's a similar check for scheduled tests). You should be able to delete the ticket though by going back to the Benchmark page -> clicking on the ticket on the right hand side -> and then from that new page should be a button to remove it.

**Comment by tstellarAMD** [comment]
I can't reconstruct all the steps I took, but I have already deleted all the tickets.  I'm sure I put it in some bad state, because I was changing the ports and manually stopping and starting the phoromatic server and the test machines while trying to get things up and running.  It's probably not worth investigating further.

It's just a test server with nothing besides my login account and a few test benchmarks, so I think I'll just delete the .db file a try again.
